### DISCUSSION TITLE
Transparency to wallpaper/blur_xray for special workspace

### DISCUSSION THREAD

**Discussion by tipa40** [hyprwm/Hyprland#1991] [Q&A]
Hi, here I go again, I would like to extend on my previous question since I marked it as solved.

I really love the blur_xray feature although my most common usage scenario would be using it for the special workspace/”scratchpad”. Unfortunately it does not seem to apply there.

I would like it for the current special workspace mode as well as being able to fullscreen special workspace focused window using blur_xray.

Is there currently a way to configure it for special workspace?
Could it be scheduled for implementation at some point?

Would really be a UI killer feature for me. :)

**Comment by vaxerski** [comment]
you can make a simple plugin for that

**Reply by tipa40** [reply]
Ah ok, thanks for the point of direction. I hope somebody will, or I will eventually :)
